src/shared/ui/button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './Button.module.css';
+import { ReactNode } from "react";
+
+type ButtonProps = {
```

Comment:
describe interfaces correctly please https://stackoverflow.com/questions/40731352/extending-html-elements-in-react-and-typescript-while-preserving-props

```ts
interface IProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
```
this interface already include what you need